src/main/webapp/bundles/common.js

```diff
@@ -0,0 +1,11 @@
+import $ from 'jquery';
+import '../lib/bootstrap';
+import 'font-awesome/css/font-awesome.css';
+import '../js/menu';
+import tooltip from '../js/tooltip';
+
+global.$ = $;
+global.jQuery = $;
+
+tooltip();
```

Comment:
hier wird noch das ein und andere Script dazu kommen, welches sich gerade auskommentiert langweilt in head.tag